Library/Homebrew/dev-cmd/tests.rb

```diff
@@ -82,6 +82,7 @@ def tests
         "--format", "ParallelTests::RSpec::RuntimeLogger",
         "--out", "tmp/parallel_runtime_rspec.log"
       ]
+      spec_args << "--tag" << "~needs_official_cmd_taps" if ENV["HOMEBREW_TEST_OFFICIAL_CMD_TAPS"].nil?
```

Comment:
So still show then as skipped when not running with `--official-cmd-taps`, and not exclude them?